docs/basics/tables.md

```diff
@@ -65,7 +65,35 @@ class MyDb(override val connector: CassandraConnection) extends Database[MyDb](c
 }
 ```
 
+### The name of the table can be controlled using `NamingStrategy`.
 
+A simple import can help you control how table names are generated. Phantom offers three variants,
+all available via `com.outworkers.phantom.NamingStrategy`. These control only the `tableName`,
+not the columns or anything else.
+
+
+```
+| Strategy                    | Casing                        | 
+| =========================== | ============================= |
+| NamingStrategy.CamelCase    | lowCamelCase                  |
+| NamingStrategy.SnakeCase    | low_snake_case                |
+| NamingStrategy.Default      | Preserves the user input      |
+```
+
+All available imports will have two flavours, and it's important to note they only work
```

Comment:
All available imports will have two flavours, and it's important to note they only work when imported in the scope where tables are defined.